bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/Activator.java

```diff
@@ -71,7 +72,11 @@ public Object addingService(ServiceReference reference) {
                 StorageService storage = (StorageService) bc.getService(reference);
                 if (storage != null) {
                     final ManagedRuleProvider rp = new ManagedRuleProvider(storage);
-                    ruleRegistry = new RuleRegistryImpl(rm, rp);
+
+                    Storage storageDisabledRules = storage.getStorage("automation_rules_disabled",
+                            this.getClass().getClassLoader());
+
+                    ruleRegistry = new RuleRegistryImpl(rm, rp, storageDisabledRules);
```

Comment:
We agree to create new issue for making RuleRegistry work without StorageService. But also in the issue Kai said "If StorageService is available, this information should be persisted in a separate Storage, otherwise it should be kept only in memory.". That means it should definitely work without. Our other registries, e.g. the ItemRegistry also work without ManagedProvider.